src/main/proto/cosmas.proto

```diff
@@ -18,6 +18,28 @@ syntax = "proto3";
 option java_package = "com.bardsoftware.papeeria.backend.cosmas";
 option java_outer_classname = "CosmasProto";
 
+message FileVersionInfo {
+    int64 generation = 1;
+    int64 timestamp = 2;
+}
+
+message FileCoffin {
+    string projectId = 1;

```

Comment:
It seems that project id is not needed here, because all coffins from the same project will be in the same protocol buffer.

Reply:
done